diff -Naru a/configure b/configure
--- a/configure	2020-07-12 22:31:41.000000000 +0200
+++ b/configure	2020-07-18 16:11:47.524587395 +0200
@@ -18443,7 +18443,7 @@
 
 
 
-ac_config_files="$ac_config_files Makefile libmp3lame/Makefile libmp3lame/i386/Makefile libmp3lame/vector/Makefile frontend/Makefile mpglib/Makefile doc/Makefile doc/html/Makefile doc/man/Makefile include/Makefile Dll/Makefile misc/Makefile dshow/Makefile ACM/Makefile ACM/ADbg/Makefile ACM/ddk/Makefile ACM/tinyxml/Makefile lame.spec mac/Makefile macosx/Makefile macosx/English.lproj/Makefile macosx/LAME.xcodeproj/Makefile vc_solution/Makefile"
+ac_config_files="$ac_config_files Makefile libmp3lame/Makefile libmp3lame/i386/Makefile libmp3lame/vector/Makefile frontend/Makefile mpglib/Makefile doc/Makefile doc/html/Makefile doc/man/Makefile include/Makefile Dll/Makefile misc/Makefile dshow/Makefile ACM/Makefile ACM/ADbg/Makefile ACM/ddk/Makefile ACM/tinyxml/Makefile lame.spec lame.pc mac/Makefile macosx/Makefile macosx/English.lproj/Makefile macosx/LAME.xcodeproj/Makefile vc_solution/Makefile"
 
 
 cat >confcache <<\_ACEOF
@@ -19500,6 +19500,7 @@
     "ACM/ddk/Makefile") CONFIG_FILES="$CONFIG_FILES ACM/ddk/Makefile" ;;
     "ACM/tinyxml/Makefile") CONFIG_FILES="$CONFIG_FILES ACM/tinyxml/Makefile" ;;
     "lame.spec") CONFIG_FILES="$CONFIG_FILES lame.spec" ;;
+    "lame.pc") CONFIG_FILES="$CONFIG_FILES lame.pc" ;;
     "mac/Makefile") CONFIG_FILES="$CONFIG_FILES mac/Makefile" ;;
     "macosx/Makefile") CONFIG_FILES="$CONFIG_FILES macosx/Makefile" ;;
     "macosx/English.lproj/Makefile") CONFIG_FILES="$CONFIG_FILES macosx/English.lproj/Makefile" ;;
diff -Naru a/configure.in b/configure.in
--- a/configure.in	2020-07-12 22:31:41.000000000 +0200
+++ b/configure.in	2020-07-18 16:09:50.033272268 +0200
@@ -1298,6 +1298,7 @@
 		 macosx/Makefile \
 		 macosx/English.lproj/Makefile \
 		 macosx/LAME.xcodeproj/Makefile \
-		 vc_solution/Makefile])
+		 vc_solution/Makefile \
+         lame.pc])
 
 AC_OUTPUT
diff -Naru a/lame.pc.in b/lame.pc.in
--- a/lame.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ b/lame.pc.in	2020-07-16 10:52:16.000000000 +0200
@@ -0,0 +1,9 @@
+prefix=@prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: @PACKAGE_NAME@
+Description: encoder that converts audio to the MP3 file format.
+Version: @VERSION@
+Libs: -L${libdir} -lmp3lame
+Cflags: -I${includedir}
diff -Naru a/Makefile.am b/Makefile.am
--- a/Makefile.am	2020-07-17 13:25:27.150968146 +0200
+++ b/Makefile.am	2020-07-17 14:19:51.973333729 +0200
@@ -27,7 +27,9 @@
 	lame.spec.in \
 	lame.spec \
 	testcase.mp3 \
-	testcase.wav
+	testcase.wav \
+	lame.pc.in \
+	lame.pc
 
 #
 # The differences depend on the used processor architecture, the used
diff -Naru a/Makefile.in b/Makefile.in
--- a/Makefile.in	2020-07-17 13:25:37.342926900 +0200
+++ b/Makefile.in	2020-07-18 16:11:27.464717644 +0200
@@ -101,7 +101,7 @@
  configure.lineno config.status.lineno
 mkinstalldirs = $(install_sh) -d
 CONFIG_HEADER = config.h
-CONFIG_CLEAN_FILES = lame.spec
+CONFIG_CLEAN_FILES = lame.spec lame.pc
 CONFIG_CLEAN_VPATH_FILES =
 AM_V_P = $(am__v_P_@AM_V@)
 am__v_P_ = $(am__v_P_@AM_DEFAULT_V@)
@@ -173,6 +173,9 @@
       && rm -rf "$(distdir)" \
       || { sleep 5 && rm -rf "$(distdir)"; }; \
   else :; fi
+am__base_list = \
+  sed '$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;s/\n/ /g' | \
+  sed '$$!N;$$!N;$$!N;$$!N;s/\n/ /g'
 am__post_remove_distdir = $(am__remove_distdir)
 am__relativize = \
   dir0=`pwd`; \
@@ -378,7 +381,14 @@
 	lame.spec.in \
 	lame.spec \
 	testcase.mp3 \
-	testcase.wav
+	testcase.wav \
+	lame.pc.in \
+	lame.pc
+
+pkgconfigexecdir = $(libdir)/pkgconfig
+pkgconfigexec_DATA = lame.pc
+
+$(pkgconfigexec_DATA): config.status
 
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
@@ -812,7 +822,7 @@
 
 install-dvi-am:
 
-install-exec-am:
+install-exec-am: install-pkgconfigexecDATA
 
 install-html: install-html-recursive
 
@@ -854,6 +864,22 @@
 
 uninstall-am:
 
+install-pkgconfigexecDATA: $(pkgconfigexec_DATA)
+	@$(NORMAL_INSTALL)
+	@list='$(pkgconfigexec_DATA)'; test -n "$(pkgconfigexecdir)" || list=; \
+	if test -n "$$list"; then \
+	  echo " $(MKDIR_P) '$(DESTDIR)$(pkgconfigexecdir)'"; \
+	  $(MKDIR_P) "$(DESTDIR)$(pkgconfigexecdir)" || exit 1; \
+	fi; \
+	for p in $$list; do \
+	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
+	  echo "$$d$$p"; \
+	done | $(am__base_list) | \
+	while read files; do \
+	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(pkgconfigexecdir)'"; \
+	  $(INSTALL_DATA) $$files "$(DESTDIR)$(pkgconfigexecdir)" || exit $$?; \
+	done
+
 .MAKE: $(am__recursive_targets) all install-am install-strip
 
 .PHONY: $(am__recursive_targets) CTAGS GTAGS TAGS all all-am \
@@ -871,7 +897,7 @@
 	installcheck installcheck-am installdirs installdirs-am \
 	maintainer-clean maintainer-clean-generic mostlyclean \
 	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
-	tags tags-am uninstall uninstall-am
+	tags tags-am uninstall uninstall-am install-pkgconfigexecDATA
 
 .PRECIOUS: Makefile
 
