--- a/configure	2022-08-31 20:58:25.000000000 +0200
+++ b/configure	2022-11-30 15:06:13.725970675 +0100
@@ -6800,6 +6800,8 @@
 rsync --help 2> /dev/null | grep -q 'contimeout' && enable rsync_contimeout || disable rsync_contimeout
 xmllint --version  > /dev/null 2>&1 && enable xmllint   || disable xmllint
 
+check_headers linux/videodev2.h
+
 # check V4L2 codecs available in the API
 if enabled v4l2_m2m; then
     check_headers linux/fb.h
